packages/adaptive-ui-explorer/src/components/control-pane/control-pane.ts

```diff
@@ -13,6 +14,9 @@ export class ControlPane extends FASTElement {
     @observable
     public showOnlyLayerBackgrounds: boolean = true;
 
+    @observable
+    public wcagContrastLevel: WcagContrastLevel = "aa";
```

Comment:
Should we consider starting to use the newer apca algorithm for contrast checks or are we waiting for wcag3.0 to be finalized before implementing?

Reply:
At some point, yes. I've recently looked into the state of that work again. I think it will be a good thing and that it will fit nicely into our algorithmic model. I've had this AA/AAA idea in my head for a while and I thought it was a good illustration of the reason to use semantic tokens. I'd like to evolve into the APCA model at a later point.